syntax = "proto3";
package tinkoff.proto;

// Request
message JsonData { string data = 1; }
